configuration/simple-topology-0.json

```diff
@@ -0,0 +1,16 @@
+{ "nodeId": 0
```

Comment:
Why is the nodeId here?

Reply:
@karknu, updated the format (`NodeSetup` doesn't have `nodeId` anymore), but forgot to remove the field from the actual file -- fixed, thanks!